packages/manager/modules/exchange/src/header/header.controller.js

```diff
@@ -129,4 +130,36 @@ export default class HeaderController {
         this.isSubmittingNewDisplayName = false;
       });
   }
+
+  removeExchangeDialog() {
+    switch (this.Exchange.value.domain.split('-')[0]) {
```

Comment:
**nitpick**
The switch could be simplified by setting the name of the hit into each case, and after call the two types of hit